test/unset.js

```diff
@@ -0,0 +1,51 @@
+'use strict'
+
+/* eslint-env mocha */
+
+const assert = require('assert')
+const unset = require('../src/unset')
+const { itEach } = require('mocha-it-each')
+
+describe('unset', () => {
+  itEach('should be able to unset an object an any arbitrary depth returning true on success',
```

Comment:
how about array of objects? e.g. path could be a[0].b.c

Reply:
I didn't think about supporting this. Should we support that?

Reply:
support for arrays added